File: Assignment 3/Files/Employee.py
from Person import Person

class Employee:
    def __init__(self, newName="none", address="none", phone="999-999-9999", department = "not assigned"):
        self.department = department
        self.name = newName

    def getDepartment(self):
        return (self.department)

    def setDepartment(self, department):
        self.department = department 

    def getName(self):
        return(self.name)

File: Assignment 4/Files/Employee.java
public class Employee extends Person{
    String name;
    String address;
    String phone;
    String department; 

    public Employee(String NewName, String address, String phone, String department){
        super(NewName, address, phone);
  
        this.department = department;

    }

    public String toString(){
        return name + " " + address + " " + phone + " " + department; 
    }

    String getDepartment(){
        return(department);
    }

    void setDepartment(String department){
        this.department = department;
    }

}
